### ISSUE TITLE
Step 39 missing IOM provider transformer child flow name

### ISSUE THREAD

**Issue by fraga** [MicrosoftDocs/dynamics-365-intelligent-order-management#4]
Step 39. Missing child flow connector name to mention "IOM Provider Transformer"

```
39. Select Add an action within the Apply to each loop, add Run a child flow from the Flow connector, and rename it "Transform Message with Power Query Online".
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 37cf6c99-4619-09f6-ad94-6d59dc574d33
* Version Independent ID: 12d529fe-2bfa-2999-daa1-fb637435ce9e
* Content: [Create intake provider](https://docs.microsoft.com/en-us/dynamics365/intelligent-order-management/lab-create-intake-provider)
* Content Source: [topics/lab-create-intake-provider.md](https://github.com/MicrosoftDocs/dynamics-365-intelligent-order-management-pr/blob/live/topics/lab-create-intake-provider.md)
* Service: **dynamics-365-intelligent-order-management**
* GitHub Login: @josaw1
* Microsoft Alias: **josaw**

**Comment by csbardella** [comment]
I ran into the same issue.  What I did was I navigated back to Flows and looked up the flow for "**IOM Provider Transformer**" and noticed that it was turned OFF so once I turned it ON I then was able to see and select the "**IOM Provider Transformer**" flow from the Child Flow drop down list. I would assume you would need to turn on any other flows required for this lab exercise. 

<img width="816" alt="image" src="https://user-images.githubusercontent.com/108139032/175567867-7af12156-d7eb-44b0-8df9-1dac839208d1.png">

Once I turned on the flow it was then visible for selection.
<img width="470" alt="image" src="https://user-images.githubusercontent.com/108139032/175569541-93c94428-7d3f-4060-91a6-7c53219b0f6e.png">
